src/options/searchInjection/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+
+import { remoteFunction } from 'src/util/webextensionRPC'
+import { Wrapper } from 'src/common-ui/components'
+
+const fetchSearchInjection = remoteFunction('fetchSearchInjection')
+const toggleSearchInjection = remoteFunction('toggleSearchInjection')
+
+class SearchInjectionContainer extends Component {
+    static propTypes = {}
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            searchInjection: true,
+        }
+        this.handleToggle = this.handleToggle.bind(this)
+    }
+
+    async componentWillMount() {
+        const searchInjection = await fetchSearchInjection()
+        this.setState({
+            searchInjection,
+        })
+    }
+
+    async handleToggle(event) {
+        await toggleSearchInjection()
+        const searchInjection = await fetchSearchInjection()
+        this.setState({
+            searchInjection,
+        })
+    }
+
+    render() {
+        return (
+            <Wrapper>
```

Comment:
No need for `<Wrapper>` here. This component renders no DOM, it's just used to allow placement of sibling JSX elements without an explicitly defined parent.